libs/components/autonumeric/src/lib/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -52,103 +52,123 @@ export class SkyAutonumericDirective
    * Assigns the name of a property from `SkyAutonumericOptionsProvider`.
    */
   @Input()
-  public set skyAutonumeric(value: SkyAutonumericOptions) {
-    this.autonumericOptions = this.mergeOptions(value);
-    this.updateAutonumericInstance();
+  public set skyAutonumeric(value: SkyAutonumericOptions | undefined) {
+    this.#autonumericOptions = this.#mergeOptions(value);
+    this.#updateAutonumericInstance();
   }
 
   /**
    * @internal
    */
   @Input()
-  public skyAutonumericFormChangesUnformatted = false;
+  public get skyAutonumericFormChangesUnformatted(): boolean | undefined {
+    return this.#_skyAutonumericFormChangesUnformatted;
+  }
+
+  public set skyAutonumericFormChangesUnformatted(value: boolean | undefined) {
+    this.#_skyAutonumericFormChangesUnformatted = value || false;
```

Comment:
A boolean value will trigger the "falsey" check of this operator. You can use null coallescing to only check against `null` or `undefined`:
```
this.#_skyAutonumericFormChangesUnformatted = value ?? false;
```

Reply:
Good suggestion. This breaks code coverage, so I'll need to write a test for passing in `undefined`.

Reply:
done in: [use null coalescing instead of logical OR for input](https://github.com/blackbaud/skyux/pull/606/commits/33851fcd14e534b5b03e409b681ff461b31e1936)

Reply:
@Blackbaud-SteveBrush does this need to be undone?